bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/discovery/HomeAssistantDiscovery.java

```diff
@@ -31,11 +31,11 @@
 import org.eclipse.smarthome.io.transport.mqtt.MqttBrokerConnection;
 import org.openhab.binding.mqtt.discovery.AbstractMQTTDiscovery;
 import org.openhab.binding.mqtt.discovery.MQTTTopicDiscoveryService;
-import org.openhab.binding.mqtt.generic.internal.MqttBindingConstants;
-import org.openhab.binding.mqtt.internal.homeassistant.BaseChannelConfiguration;
-import org.openhab.binding.mqtt.internal.homeassistant.ChannelConfigurationTypeAdapterFactory;
-import org.openhab.binding.mqtt.internal.homeassistant.HaID;
-import org.openhab.binding.mqtt.internal.homeassistant.HandlerConfiguration;
+import org.openhab.binding.mqtt.homeassistant.generic.internal.MqttBindingConstants;
+import org.openhab.binding.mqtt.homeassistant.internal.homeassistant.BaseChannelConfiguration;
+import org.openhab.binding.mqtt.homeassistant.internal.homeassistant.ChannelConfigurationTypeAdapterFactory;
```

Comment:
You might leave all "internal" classes where they are (doesn't look good and doesn't make sense: "homeassistant.internal.homeassistant").

If there are any interferences, they are caused by exposed/non-internal classes

Reply:
Done!